src/main/kotlin/uk/gov/justice/digital/hmpps/pecs/jpc/controller/OutputSpreadsheetController.kt

```diff
@@ -37,11 +37,9 @@ class OutputSpreadsheetController(
     response: HttpServletResponse?,
     authentication: Authentication?
   ): ResponseEntity<InputStreamResource?>? {
-    auditService.create(AuditableEvent.createDownloadSpreadsheetEvent(movesFrom, supplier, authentication!!))
-
-    return spreadsheetService.spreadsheet(supplier, movesFrom)?.let { file ->
+    return spreadsheetService.spreadsheet(authentication!!, Supplier.valueOfCaseInsensitive(supplier), movesFrom)?.let { file ->
```

Comment:
This ensures we have a valid Supplier. Tthe value of UNKNOWN can be returned as well, probably OK for now.